### PR TITLE
RUBY-879: Implement minHeartbeatFrequency at 500ms

### PR DESCRIPTION
**Pull request by durran**
https://jira.mongodb.org/browse/RUBY-879

### CODE REVIEW

**Comment by estolfo** [comment]
lgtm, I'm glad we're all on the same page re this now = )

**Comment by ajdavis** [comment]
LGTM.

File: lib/mongo/server/monitor.rb [hunk 1]
```diff
@@ -29,6 +29,12 @@ class Monitor
       # @since 2.0.0
       HEARTBEAT_FREQUENCY = 10.freeze
 
+      # The minimum time between forced server scans. Is
+      # minHeartbeatFrequencyMS in the SDAM spec.
+      #
+      # @since 2.0.0
+      MIN_SCAN_FREQUENCY = 0.5.freeze
```
Comment (arthurnn) [inline lib/mongo/server/monitor.rb]: why are you `freeze` this constant?

Comment (durran) [inline lib/mongo/server/monitor.rb]: Just habit - doesn't do anything on numbers but doesn't have any negative effects either.

**Pull request merged**